pkg/konfluxgen/releaseplanadmission.template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  name: {{{ truncate ( sanitize .ApplicationName ) }}}
+  namespace: rhtap-releng-tenant
+spec:
+  applications: [{{{ truncate ( sanitize .ApplicationName ) }}}]
+  origin: ocp-serverless-tenant
+  policy: registry-standard
+  data:
+    mapping:
+      components:
+      {{{- range $component := .Components }}}
+        - name: {{{ $component.ComponentName }}}
+          repository: {{{ $component.ImageRepository }}}
+          pushSourceContainer: false
```

Comment:
why false? Pushing source images can help for disaster recovery

Reply:
I wasn't sure if we need them in the production registry. Let me change it then...

Reply:
31acfba6